### ISSUE TITLE
Tag values should allow for spaces

### ISSUE THREAD

**Issue by esatterwhite** [node-influx/node-influx#120]
I'm unable to set tag values if they have sapces, the engine fails to parse the query.

Is there something I should be doing to allow for spaces?

**Comment by esatterwhite** [comment]
```js
client.writePoint('sample'
    , {attr:1001, time: new Date('2014-03-01T12:01:01') }
    , {customer:1, movement:"hello world"}
    , console.log
)
```

**Comment by mauvm** [comment]
Which version of node-influx are you using? 3.* or 4.*?

**Comment by esatterwhite** [comment]
4

**Comment by mauvm** [comment]
Try adding a `console.log(requestOptions)` on line 109 of https://github.com/node-influx/node-influx/blob/master/lib/InfluxRequest.js and check if the send body contains double quotes around the `movement` attribute.

**Comment by mauvm** [comment]
Oops, I thought you were inserting it as attribute instead of a tag. In any case, to answer your question: no, it should work by default. Also see https://github.com/node-influx/node-influx/blob/master/index.js#L262.

I found the problem though: the changes in this repository have not yet been published and are not in 4.0.1.

**Comment by bencevans** [comment]
We've pushed a new release! Checkout `v4.1.0`.
